std/numeric.d

```diff
@@ -1382,6 +1382,234 @@ unittest
     static assert(__traits(compiles, findRoot((real x)=>cast(double)x, real.init, real.init)));
 }
 
+/++
+Find a real minimum of a real function $(D f(x)) via bracketing.
+Given a function $(D f) and a range $(D (ax..bx)),
+returns the value of $(D x) in the range which is closest to a minimum of $(D f(x)).
+$(D f) is never evaluted at the endpoints of $(D ax) and $(D bx).
+If $(D f(x)) has more than one minimum in the range, one will be chosen arbitrarily.
+If $(D f(x)) returns NaN or -Infinity, $(D (x, f(x), NaN)) will be returned;
+otherwise, this algorithm is guaranteed to succeed.
+
+Params:
+    f = Function to be analyzed
+    ax = Left bound of initial range of f known to contain the minimum.
+    bx = Right bound of initial range of f known to contain the minimum.
+    relTolerance = Relative tolerance.
+    absTolerance = Absolute tolerance.
+
+Preconditions:
+    $(D ax) and $(D bx) shall be finite reals. $(BR)
+    $(D relTolerance) shall be normal positive real. $(BR)
+    $(D absTolerance) shall be normal positive real no less then $(D T.epsilon*2).
+
+Returns:
+    A tuple consisting of $(D x), $(D y = f(x)) and $(D error = 3 * (absTolerance * fabs(x) + relTolerance)).
+
+    The method used is a combination of golden section search and
+successive parabolic interpolation. Convergence is never much slower
+than that for a Fibonacci search.
+
+References:
+    "Algorithms for Minimization without Derivatives", Richard Brent, Prentice-Hall, Inc. (1973)
```

Comment:
A link would be nice. Is https://en.wikipedia.org/wiki/Brent%27s_method appropriate?